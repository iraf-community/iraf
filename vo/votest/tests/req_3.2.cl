#
#  Access every record in a Registry result record.

# Set the test description string.
votest.descr = "Access every record in a Registry result record"

print ("------------------------------------------------------------------")
print ("Req  3.3:  Users shall have access to all information in the query")
print ("	   return record.")
print ("------------------------------------------------------------------")


fcache init

# -------------------------------------------
# Execute the test commands.
# -------------------------------------------


# -------------------------------------------
# Using the REGISTRY task we can print all 
# fields in the record as follows:
# -------------------------------------------

registry ("noao", verbose+, record=2, interactive-)
printf ("\n\n")


# -------------------------------------------
# Programmatically this is done as follows:
# -------------------------------------------
int   ires, icount

ires = regSearch ("noao", 0)
icount = regResCount (ires)

printf ("\nFound %d records:\n\n", icount)

printf ("Num  %10.10s  %s\n", "ShortName", "Title")
printf ("---  %10.10s  %s\n", "---------", "-----")

for (i=0; i < icount; i=i+1) {
    printf ("%3d  %10.10s  %s\n", (i+1), 
	trim(regValue (ires, "ShortName", i)),
	trim(regValue (ires, "Title", i))
}
