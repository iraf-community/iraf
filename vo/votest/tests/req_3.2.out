------------------------------------------------------------------
Req  3.3:  Users shall have access to all information in the query
	   return record.
------------------------------------------------------------------
#
#  Record: 2
#
#  Registry Query Constraints:
#	  Query Term: 'chandra'
#	 ServiceType:  image
#	    Bandpass:  x-ray
#
          Title: Chandra Transmission Grating Catalog and Archive, Simple 
                 Image Access Interface
      ShortName: TGCat SIA
     Identifier: ivo://cxc.mit/tgcat/SIA#1
     ServiceURL: http://tgcat.mit.edu/tgSia.php?
   ReferenceURL: 
    Description: This resource describes the Simple Image Access interface 
                 provided by TGCat. References returned are of three types: 
                 1) evt2 files - for ACIS see: http:// 
                 This resource describes the Simple Image Access interface provided by TGCat. References returned are of three types: 1) evt2 files - for ACIS see: http:// xc.harvard.edu/ciao/data_products_guide/event_descrip.html#acisl2tgevts - for HRC see: http://c c.h
        Subject: x ray spectra,transmission grating,gratings,spectroscopy,
                 high resolution spectroscopy,X RAY SPECTROSCOPY,HETG,LETG,
                 HEG,MEG,LEG,ACIS,HRC,Chandra
   ResourceType: SimpleImageAccess
           Type: 
        Creator: Chandra Transmission Grating Catalog and Archive
       Facility: 
     Instrument: 
    Contributor: 
   CoverageSpat: 
   CoverageTemp: 
   CoverageSpec: X-ray
   ContentLevel: Research
        Version: 1
        Version: 1
        Version: 1
        COLUMNS: 
           dbid: 
         status: 
     Identifier: ivo://cxc.mit/tgcat/SIA#1
          Title: Chandra Transmission Grating Catalog and Archive, Simple 
                 Image Access Interface
      ShortName: TGCat SIA
   CurationPubl: 
   CurationPubl: 
   CurationPubl: 
   CurationPubl: 
   CurationCrea: 
   CurationCrea: 
   CurationCont: 
   CurationDate: 
   CurationVers: 
   CurationCont: 
   CurationCont: 
   CurationCont: 
   CurationCont: 
        Subject: x ray spectra,transmission grating,gratings,spectroscopy,
                 high resolution spectroscopy,X RAY SPECTROSCOPY,HETG,LETG,
                 HEG,MEG,LEG,ACIS,HRC,Chandra
    Description: This resource describes the Simple Image Access interface 
                 provided by TGCat. References returned are of three types: 
                 1) evt2 files - for ACIS see: http:// 
                 This resource describes the Simple Image Access interface provided by TGCat. References returned are of three types: 1) evt2 files - for ACIS see: http:// xc.harvard.edu/ciao/data_products_guide/event_descrip.html#acisl2tgevts - for HRC see: http://c c.h
   ReferenceURL: 
           Type: 
       Facility: 
     Instrument: 
   ContentLevel: Research
   Modification: 
     ServiceURL: http://tgcat.mit.edu/tgSia.php?
   CoverageSpat: 
   CoverageSpec: X-ray
   CoverageTemp: 
   CoverageRegi: 
   ResourceType: SimpleImageAccess
            xml: 
   harvestedfro: 
   harvestedfro: 
      footprint: 
   validationLe: 




Found 146 records:

Num   ShortName  Title
---   ---------  -----
  1       NDWFS  NOAO Deep Wide-Field Survey
  2  J/AJ/128/1  NOAO Fundamental Plane Survey (Smith+, 2004)
  3  J/AJ/128/1  NOAO Fundamental Plane Survey (Smith+, 2004)
  4  J/AJ/128/1  NOAO Fundamental Plane Survey (Smith+, 2004)
  5  J/AJ/128/1  NOAO Fundamental Plane Survey (Smith+, 2004)
  6  J/AJ/128/1  First Look Survey: NOAO R-band Mosaic (Fadda+, 2004)
  7  J/AJ/128/1  First Look Survey: NOAO R-band Mosaic (Fadda+, 2004)
  8  J/A+A/535/  NOAO Bootes field 153MHz observations (Intema+, 2011)
  9  J/A+A/535/  NOAO Bootes field 153MHz observations (Intema+, 2011)
 10  J/A+A/535/  NOAO Bootes field 153MHz observations (Intema+, 2011)
 11  FLS_ELAISN  Spitzer First Look Survey (FLS) -- NOAO ELAIS N1 -- R
 12  FLS_MAIN_R  Spitzer First Look Survey (FLS) -- NOAO Extragalactic -- R
 13  NOAO ARCH   NOAO Archive Authority
 14  NOAO VOEve  NOAO VOEvent Authority
 15  NOAO NVO N  NOAO NVO Service Authority
 16  J/ApJ/634/  16{mu}m sources in the NOAO Bootes field (Kasliwal+, 2005)
 17  J/ApJ/634/  16{mu}m sources in the NOAO Bootes field (Kasliwal+, 2005)
 18  J/ApJ/634/  16{mu}m sources in the NOAO Bootes field (Kasliwal+, 2005)
 19  BOOF153MHZ  Bootes Field GMRT 153-MHz Source Catalog
 20  BOOF153MHZ  Bootes Field GMRT 153-MHz Source Catalog
 21  BOOF153MHZ  Bootes Field GMRT 153-MHz Source Catalog
 22  J/ApJ/684/  z&gt 1 galaxy clusters from IRAC Shallow Survey (Eisenhardt+, 2008)
 23  J/ApJ/684/  z&gt 1 galaxy clusters from IRAC Shallow Survey (Eisenhardt+, 2008)
 24  J/ApJ/684/  z&gt 1 galaxy clusters from IRAC Shallow Survey (Eisenhardt+, 2008)
 25  J/ApJ/684/  z&gt 1 galaxy clusters from IRAC Shallow Survey (Eisenhardt+, 2008)
 26  J/MNRAS/36  FSVS Cluster Catalogue (Sochting+, 2006)
 27  J/MNRAS/36  FSVS Cluster Catalogue (Sochting+, 2006)
 28  J/MNRAS/36  FSVS Cluster Catalogue (Sochting+, 2006)
 29  J/MNRAS/36  FSVS Cluster Catalogue (Sochting+, 2006)
 30  J/MNRAS/36  FSVS Cluster Catalogue (Sochting+, 2006)
 31  J/AJ/126/2  CL 0024+1654 UBVI photometry (Alexov+, 2003)
 32  J/AJ/126/2  CL 0024+1654 UBVI photometry (Alexov+, 2003)
 33  J/AJ/136/1  Open clusters as galactic disk tracers. I. (Frinchaboy+, 2008)
 34  J/AJ/136/1  Open clusters as galactic disk tracers. I. (Frinchaboy+, 2008)
 35  J/AJ/136/1  Open clusters as galactic disk tracers. I. (Frinchaboy+, 2008)
 36  J/AJ/136/1  Open clusters as galactic disk tracers. I. (Frinchaboy+, 2008)
 37  J/AJ/136/1  Open clusters as galactic disk tracers. I. (Frinchaboy+, 2008)
 38  J/AJ/133/2  Star beyond the NLTT catalog (Reid+, 2007)
 39  J/AJ/133/2  Star beyond the NLTT catalog (Reid+, 2007)
 40  J/AJ/133/2  Star beyond the NLTT catalog (Reid+, 2007)
 41  J/AJ/133/2  Star beyond the NLTT catalog (Reid+, 2007)
 42  J/ApJ/676/  Spectral templates for galaxies from 0.2 to 10um (Assef+, 2008)
 43  J/ApJ/676/  Spectral templates for galaxies from 0.2 to 10um (Assef+, 2008)
 44  J/ApJS/191  Southern Cosmology Survey. II. (Menanteau+, 2010)
 45  J/ApJS/191  Southern Cosmology Survey. II. (Menanteau+, 2010)
 46  J/ApJS/191  Southern Cosmology Survey. II. (Menanteau+, 2010)
 47  J/ApJS/191  Southern Cosmology Survey. II. (Menanteau+, 2010)
 48  J/ApJS/191  Southern Cosmology Survey. II. (Menanteau+, 2010)
 49  J/ApJS/191  Southern Cosmology Survey. II. (Menanteau+, 2010)
 50  J/ApJS/185  58 radio sources near BNGSs (Stalder+, 2009)
 51  J/ApJS/185  58 radio sources near BNGSs (Stalder+, 2009)
 52  J/ApJS/185  58 radio sources near BNGSs (Stalder+, 2009)
 53  J/ApJ/645/  ChaMP serendipitous galaxy cluster survey (Barkhouse+, 2006)
 54  J/ApJ/645/  ChaMP serendipitous galaxy cluster survey (Barkhouse+, 2006)
 55  J/ApJ/645/  ChaMP serendipitous galaxy cluster survey (Barkhouse+, 2006)
 56  J/ApJ/645/  ChaMP serendipitous galaxy cluster survey (Barkhouse+, 2006)
 57  J/A+A/453/  Line-strength indices in IC 4200 (Serra+, 2006)
 58  J/A+A/453/  Line-strength indices in IC 4200 (Serra+, 2006)
 59  J/AJ/127/2  LALA Bootes field X-ray source catalog (Wang+, 2004)
 60  J/AJ/127/2  LALA Bootes field X-ray source catalog (Wang+, 2004)
 61  J/PASP/114  Stellar variability in field stars (Everett+, 2002)
 62  J/PASP/114  Stellar variability in field stars (Everett+, 2002)
 63  J/PASP/114  Stellar variability in field stars (Everett+, 2002)
 64  J/AJ/124/6  Absolute positions and proper motions in NGC 188 (Platais+, 2002)
 65  J/AJ/124/6  Absolute positions and proper motions in NGC 188 (Platais+, 2002)
 66  J/AJ/124/6  Absolute positions and proper motions in NGC 188 (Platais+, 2002)
 67  J/A+A/384/  BVRIHalpha photometry of NGC 2244 X-ray stars (Berghoefer+, 2002)
 68  J/A+A/384/  BVRIHalpha photometry of NGC 2244 X-ray stars (Berghoefer+, 2002)
 69  J/A+A/384/  BVRIHalpha photometry of NGC 2244 X-ray stars (Berghoefer+, 2002)
 70  XBOOTESOID  XBOOTES:NDWFSBootesFieldOptical&amp NearIRCounterparts
 71  XBOOTESOID  XBOOTES:NDWFSBootesFieldOptical&amp NearIRCounterparts
 72  XBOOTESOID  XBOOTES:NDWFSBootesFieldOptical&amp NearIRCounterparts
 73  J/ApJ/701/  5000 AGNs behind the Magellanic clouds (Kozlowski+, 2009)
 74  J/ApJ/701/  5000 AGNs behind the Magellanic clouds (Kozlowski+, 2009)
 75  J/ApJ/701/  5000 AGNs behind the Magellanic clouds (Kozlowski+, 2009)
 76  J/MNRAS/38  Velocities and photometry in NGC 2112 (Carraro+, 2008)
 77  J/MNRAS/38  Velocities and photometry in NGC 2112 (Carraro+, 2008)
 78  J/MNRAS/38  Velocities and photometry in NGC 2112 (Carraro+, 2008)
 79  J/MNRAS/38  Triple and quadruple stars (Tokovinin, 2008)
 80  J/MNRAS/38  Triple and quadruple stars (Tokovinin, 2008)
 81  J/MNRAS/38  Triple and quadruple stars (Tokovinin, 2008)
 82  J/MNRAS/38  Triple and quadruple stars (Tokovinin, 2008)
 83  J/ApJ/614/  Planetary nebulae in M33 (Ciardullo+, 2004)
 84  J/ApJ/614/  Planetary nebulae in M33 (Ciardullo+, 2004)
 85  J/ApJ/614/  Planetary nebulae in M33 (Ciardullo+, 2004)
 86  J/ApJ/706/  LAEs at z~4.5 in the LALA Cetus field (Wang+, 2009)
 87  J/ApJ/706/  LAEs at z~4.5 in the LALA Cetus field (Wang+, 2009)
 88  J/ApJ/706/  LAEs at z~4.5 in the LALA Cetus field (Wang+, 2009)
 89  PIGSSBOOFD  Allen Telescope Array Pi GHz Sky Survey (PiGSS) Boo Field Source Catalog
 90  PIGSSBOOFD  Allen Telescope Array Pi GHz Sky Survey (PiGSS) Boo Field Source Catalog
 91  PIGSSBOOFD  Allen Telescope Array Pi GHz Sky Survey (PiGSS) Boo Field Source Catalog
 92  LALABOOCXO  LALA Bootes Field Chandra X-Ray Point Source Catalog
 93  LALABOOCXO  LALA Bootes Field Chandra X-Ray Point Source Catalog
 94  LALABOOCXO  LALA Bootes Field Chandra X-Ray Point Source Catalog
 95     XBootes  XBOOTES: NDWFS Bootes Field X-Ray Point Source Catalog
 96     XBootes  XBOOTES: NDWFS Bootes Field X-Ray Point Source Catalog
 97     XBootes  XBOOTES: NDWFS Bootes Field X-Ray Point Source Catalog
 98  J/MNRAS/38  BVRcIc photometry of NGC 821 globular clusters (Spitler+, 2008)
 99  J/MNRAS/38  BVRcIc photometry of NGC 821 globular clusters (Spitler+, 2008)
100  J/MNRAS/38  BVRcIc photometry of NGC 821 globular clusters (Spitler+, 2008)
101  J/AJ/127/2  Early-type galaxies color evolution (Holden+, 2004)
102  J/AJ/127/2  Early-type galaxies color evolution (Holden+, 2004)
103  J/ApJ/619/  Catalog of nearby cluster galaxy photometry (McIntosh+, 2005)
104  J/ApJ/619/  Catalog of nearby cluster galaxy photometry (McIntosh+, 2005)
105  J/ApJ/619/  Catalog of nearby cluster galaxy photometry (McIntosh+, 2005)
106       GOODS  The Great Observatories Origins Deep Survey
107  J/AJ/117/5  1997 WIYN binary stars speckle observations (Horch+, 1999)
108  J/AJ/117/5  1997 WIYN binary stars speckle observations (Horch+, 1999)
109  J/AJ/124/3  uz,vz,bz,yz photometry in Coma cluster (Odell+, 2002)
110  J/AJ/124/3  uz,vz,bz,yz photometry in Coma cluster (Odell+, 2002)
111  J/AJ/124/3  uz,vz,bz,yz photometry in Coma cluster (Odell+, 2002)
112  J/AJ/124/3  uz,vz,bz,yz photometry in Coma cluster (Odell+, 2002)
113  J/AJ/124/3  uz,vz,bz,yz photometry in Coma cluster (Odell+, 2002)
114  J/AJ/131/2  Optical and radio study of Abell 2111 (Miller+, 2006)
115  J/AJ/131/2  Optical and radio study of Abell 2111 (Miller+, 2006)
116  J/AJ/131/2  Optical and radio study of Abell 2111 (Miller+, 2006)
117  J/AJ/131/2  Optical and radio study of Abell 2111 (Miller+, 2006)
118  J/AJ/131/2  Optical and radio study of Abell 2111 (Miller+, 2006)
119  J/AJ/133/1  Stromgren photometry in NGC 6791 (Anthony-Twarog+, 2007)
120  J/AJ/133/1  Stromgren photometry in NGC 6791 (Anthony-Twarog+, 2007)
121  J/AJ/133/1  Stromgren photometry in NGC 6791 (Anthony-Twarog+, 2007)
122  J/MNRAS/39  Lyman{alpha} forest in QSO pairs (Kirkman+, 2008)
123  J/MNRAS/39  Lyman{alpha} forest in QSO pairs (Kirkman+, 2008)
124  J/MNRAS/39  Lyman{alpha} forest in QSO pairs (Kirkman+, 2008)
125  J/AJ/129/8  Stromgren photometry of NGC 2243 (Anthony-Twarog+, 2005)
126  J/AJ/129/8  Stromgren photometry of NGC 2243 (Anthony-Twarog+, 2005)
127  J/AJ/129/8  Stromgren photometry of NGC 2243 (Anthony-Twarog+, 2005)
128  J/AJ/121/2  3{mu}m survey of the Chamaeleon I dark cloud (Kenyon+, 2001)
129  J/AJ/121/2  3{mu}m survey of the Chamaeleon I dark cloud (Kenyon+, 2001)
130  J/AJ/121/2  3{mu}m survey of the Chamaeleon I dark cloud (Kenyon+, 2001)
131  J/AJ/111/1  JHK photometry of NGC1333 (Lada+, 1996)
132  J/AJ/111/1  JHK photometry of NGC1333 (Lada+, 1996)
133  J/AJ/111/1  JHK photometry of NGC1333 (Lada+, 1996)
134         DLS  Deep Lens Survey
135  J/MNRAS/30  HST VI Photometry of Six LMC Old Globular Clusters (Olsen+ 1998)
136  J/MNRAS/30  HST VI Photometry of Six LMC Old Globular Clusters (Olsen+ 1998)
137  J/ApJ/562/  Old Stellar Populations of the SMC (Dolphin+ 2001)
138  J/ApJ/562/  Old Stellar Populations of the SMC (Dolphin+ 2001)
139  J/ApJS/152  Indo-US library of coude feed stellar spectra (Valdes+, 2004)
140  J/ApJS/152  Indo-US library of coude feed stellar spectra (Valdes+, 2004)
141    STScIReg  STScI Searchable Registry
142    STScIReg  STScI Searchable Registry
143  Landolt198  UBVRI Photometric Standard Stars Around the Celest
144  Landolt199  UBVRI Photometric Standard Stars in the Magnitude
145  USNO-SA2.0  USNO-SA2.0
146   USNO-A2.0  USNO-A2.0
